### PR TITLE
Added Email warning

### PR DESCRIPTION
**Pull request by ankitgadling**
#26

### CODE REVIEW

File: app/users/views.py [hunk 1]
```diff
@@ -36,15 +36,23 @@ def get_magic_link(request, user):
...
 
-        You requested that we send you a link to log in to our app:
+        You've requested a secure login link to access our app:
 
-            {link}
+        **IMPORTANT SECURITY REMINDERS:**
 
-        Thank you for using django-sesame!
+        - Please treat this link like a secret key. Do not share it with anyone.
+        - Our administrators will **never** ask you to reveal or share your magic login link.
```
Comment (iokiwi) [inline app/users/views.py:48]: Flake8 is warning us that this line is too long

File: app/users/views.py [hunk 2]
```diff
@@ -36,15 +36,23 @@ def get_magic_link(request, user):
 def send_email(user, link):
     """Send an email with this login link to this user."""
     user.email_user(
-        subject="[django-sesame] Log in to our app",
+        subject="[MoneyApp] Log in to our app",
```
Comment (iokiwi) [inline app/users/views.py:39]: Love this! Thank you ❤️

File: app/users/views.py [hunk 3]
```diff
@@ -36,15 +36,23 @@ def get_magic_link(request, user):
...
-        Thank you for using django-sesame!
+        - Please treat this link like a secret key. Do not share it with anyone.
+        - Our administrators will **never** ask you to reveal or share your magic login link.
+
+        To log in, simply click the following link: {link}
+
+        Your online security and privacy are our top priorities.
+
+        Best regards,
+        The MoneyApp Team
```
Comment (iokiwi) [inline app/users/views.py:55]: I love you're attention to detail here :)

**Review by iokiwi — APPROVED** [review]

**Comment by iokiwi** [comment]
For future PR's make sure to run flake8 against your code before committing : )

File: app/users/views.py (hunk 3 above)
Comment (ankitgadling) [reply app/users/views.py:55]: thank you

**Comment by ankitgadling** [comment]
Yes I will make sure to run flake8 in future PR's

**Pull request merged**